src/features/session_replay/instrument/index.js

```diff
@@ -44,10 +54,15 @@ export class Instrument extends InstrumentBase {
   }
 
   async #startRecording (mode) {
+    if (this.featAggregate) {
+      if (this.featAggregate.entitled) this.featAggregate.initializeRecording(mode, true)
+      /** dont need to reimport the recorder so just return if the featAgg exists */
+      return
+    }
+    if (this.recorder?.recording) return
```

Comment:
Here's some case flows:
Ex. A

1. api recordReplay called -> import Recorder, fetching
2. preload evaluated -> quits because a startup already in process -> queues agg import
3. Recorder imported & starts recording
4. agg imports

Ex. B

1. preload evaluated -> import Recorder, fetching
2. api recordReplay called -> quits because a startup already in process
3. Recorder imported & starts recording -> queues agg import
4. agg imports

Similar scenario for if preload evaluated to false, just that recordReplay will do the startup.